worldedit-core/src/main/java/com/sk89q/worldedit/WorldEdit.java

```diff
@@ -240,6 +242,15 @@ public TranslationManager getTranslationManager() {
         return translationManager;
     }
 
+    /**
+     * Return the image brush manager.
+     *
+     * @return the image brush manager
+     */
+    public ImageBrushManager getImageBrushManager() {
```

Comment:
I'm unsure whether this should be specifically an ImageBrushManager

A more generic ImageManager, or maybe even a manager for generic assets that could be used in WE stuff (like models etc) could make more sense.

Reply:
Did _something_, but sure if the changes are close to what you imagined :p